tests/server.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
I used this script to compare the behavior of Chrome and Firefox with this lib. I think it can be useful but I can remove it you prefer.